src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import type {ImageStyle} from 'react-native'
```

Comment:
duplicate import 'react-native' here

Reply:
The line at row 2 is only import type. I want to separate the import type and the import component into different lines